visualization/motion_planning_rviz_plugin/src/ui/motion_planning_rviz_plugin_frame.ui

```diff
@@ -628,7 +628,31 @@
              </widget>
             </item>
             <item>
-             <widget class="QDoubleSpinBox" name="exec_time_factor">
+             <widget class="QDoubleSpinBox" name="velocity_scaling_factor">
+              <property name="maximum">
+               <double>1.000000000000000</double>
+              </property>
+              <property name="singleStep">
+               <double>0.010000000000000</double>
+              </property>
+              <property name="value">
+               <double>1.000000000000000</double>
+              </property>
+             </widget>
+            </item>
+           </layout>
+          </item>
+          <item>
+           <layout class="QHBoxLayout" name="horizontalLayout_20">
+            <item>
+             <widget class="QLabel" name="label_20">
+              <property name="text">
+               <string>Acceleration Scaling:</string>
```

Comment:
I agree with you and thing the whole widget is a bit big.  As for the addition, I can do a side by side comparison but I am pretty sure that the widget width went unchanged.